packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -96,6 +99,8 @@ const Dashboard = ({
         }
     };
 
+    const is_tour_complete = React.useRef(true);
+
     let getBotBuilderToken: string | number = '';
```

Comment:
@farabi-deriv `get` and `set` prefix should be used only in case of methods/functions
I believe we can change it to `bot_builder_token` or `bot_builder_tour_token`
and `onboarding_token` or `onboarding_tour_token` as per our variable naming conventions.

In this way it is more meaningful and self explanatory.

Reply:
alright sure, done ✅